### ISSUE TITLE
overpass-turbo.eu is broken returning this error for all queries: "runtime error: The dispatcher (i.e. the database management system) is turned off."

### ISSUE THREAD

**Issue by alaurenz** [tyrasd/overpass-turbo#629]
<img width="933" alt="Screen Shot 2023-04-05 at 9 13 21 PM" src="https://user-images.githubusercontent.com/303765/230248069-32e46927-eef1-4852-a2b3-fc5fe6816ad9.png">

**Comment by mmd-osm** [comment]
You need to report it to whatever Overpass server you’re using. It’s outside of overpass turbo control and off topic for this repo.

**Comment by alaurenz** [comment]
@mmd-osm Thank you for clarifying that, I did not realize multiple servers could be toggled in the user interface. I was able to get things working by changing to another server.

For those who ay find this helpful, see this screenshot for how to toggle the Overpass server. You go to Settings -> toggle Server dropdown:
<img width="605" alt="Screen Shot 2023-04-11 at 5 27 44 PM" src="https://user-images.githubusercontent.com/303765/231292185-9b379875-2f81-4a37-9b98-c07252daed8d.png">
